htdocs/stancerdolicloud/class/actions_stancerdolicloud.class.php

```diff
@@ -558,8 +558,11 @@ public function isPaymentOK($parameters, &$object, &$action, $hookmanager)
 	{
 		global $langs;
 
+		require_once DOL_DOCUMENT_ROOT."/core/lib/geturl.lib.php";
+		include_once DOL_DOCUMENT_ROOT.'/core/lib/security.lib.php';
+
 		$error = 0; // Error counter
-		$ispaymentok = false;
+		$ispaymentok = true;
```

Comment:
Using a default validation to true and testing if api result is ko is not secured.
Can't you invert the logic:
Must be false by default and confirmed if the api call return the correct information.